build.gradle

```diff
@@ -27,7 +27,7 @@ plugins {
 group = 'com.linkedin.avroutil1'
 
 //does not include "helper" as that module has its own publishing section (because fat jar)
```

Comment:
Can we leave another comment here to say why fastserde can't be treated via the default process?